observability/otel/otel-configuring-metrics-for-monitoring-stack.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="otel-configuring-metrics-for-monitoring-stack"]
+= Configuring metrics for the monitoring stack
+include::_attributes/common-attributes.adoc[]
+:context: otel-configuring-metrics-for-monitoring-stack
+
+toc::[]
+
+As a cluster administrator, you can configure the OpenTelemetry Collector custom resource (CR) to perform the following tasks:
+
+* Create a Prometheus `ServiceMonitor` CR for scraping the Collector’s pipeline metrics and the enabled Prometheus exporters.
+
+* Configure the Prometheus receiver to scrape metrics from the in-cluster monitoring stack.
+
+include::modules/otel-config-send-metrics-monitoring-stack.adoc[leveloffset=+1]
+include::modules/otel-config-receive-metrics-monitoring-stack.adoc[leveloffset=+1]
+
+[id="additional-resources_otel-configuring-metrics-for-monitoring-stack"]
+== Additional resources
+
+// * xref:../monitoring/accessing-third-party-monitoring-apis.adoc#monitoring-querying-metrics-by-using-the-federation-endpoint-for-prometheus[Querying metrics by using the federation endpoint for Prometheus]
+
+//* xref:../monitoring/accessing-third-party-monitoring-apis.adoc#monitoring-querying-metrics-by-using-the-federation-endpoint-for-prometheus_accessing-third-party-monitoring-apis[Querying metrics by using the federation endpoint for Prometheus]
+
+* xref:../monitoring/accessing-third-party-monitoring-apis.adoc#monitoring-querying-metrics-by-using-the-federation-endpoint-for-prometheus[Querying metrics by using the federation endpoint for Prometheus]
```

Comment:
This xref path is incorrect. I think you're missing the trailing `{context}` value. 

```suggestion
* xref:../monitoring/accessing-third-party-monitoring-apis.adoc#monitoring-querying-metrics-by-using-the-federation-endpoint-for-prometheus_accessing-monitoring-apis-by-using-the-cli[Querying metrics by using the federation endpoint for Prometheus]
```

Reply:
Thank you. I managed to fix it.